webcomponents/vectortile-view-component/angular.json

```diff
@@ -24,6 +24,11 @@
                 "glob": "**/*",
                 "input": "src/assets/teststyles",
                 "output": "styles"
+              },
+              {
+                "glob": "**/*",
+                "input": "src/assets/testdata",
+                "output": "testdata"
```

Comment:
houden we testdata buiten het uiteindelijke artifact/bundle? Niet dat testdata in productie code terecht komt.

Reply:
Goed punt, assets leeg gemaakt. Test data hoort niet thuis in code.  Testdata zit wel in cypress omgeving (fixtures , developer dependencies)  zodat tests lokaal kunnen draaien.